ghci-script-mode.el

```diff
@@ -52,7 +52,9 @@
 ;;;###autoload
 (add-to-list 'auto-mode-alist '("\\.ghci\\'" . ghci-script-mode))
 
-(define-key ghci-script-mode-map (kbd "C-c C-l") 'ghci-script-mode-load)
+(easy-mmode-defmap ghci-script-mode-map
+  `("C-c C-l" . ghci-script-mode-load)
```

Comment:
Oh, I suppose it's not enough. Likely it should be:

`((kbd "C-c C-l") . #'ghci-script-mode-load)`.

`#'` explanation: http://endlessparentheses.com/get-in-the-habit-of-using-sharp-quote.html